packages/core/src/odata/common/response-data-accessor.ts

```diff
@@ -0,0 +1,7 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+export interface ResponseDataAccessor {
+  getCollectionResult: (data) => any[];
+  isCollectionResult: (data) => boolean;
+  getSingleResult: (data: any) => Record<string, any>;
```

Comment:
The `getLinkedCollectionResult` is now also in the `response-data-accessor` files. Should one include it?